recipes/qtxlsxwriter/all/conandata.yml

```diff
@@ -0,0 +1,10 @@
+sources:
+  "0.3.0":
+    - url: "https://github.com/dbzhang800/QtXlsxWriter/archive/v0.3.0.zip"
+      sha256: "e665317de4f1551936d519781ab8f96b6738926595f3f26ecb6c50db4ff0419e"
+    - url: "https://raw.githubusercontent.com/dbzhang800/QtXlsxWriter/master/LICENSE"
```

Comment:
It will be helpful to have the license included as part of the release. Could we ask the author about this? I think it is a good practice and would be useful

Reply:
It's already added to master so it'll be there (presumably) https://github.com/conan-io/conan-center-index/pull/4838#discussion_r594235489